packages/cli/src/errors/missing-execution-stop.error.ts

```diff
@@ -1,6 +1,6 @@
-import { ApplicationError } from 'n8n-workflow';
+import { UnexpectedError } from 'n8n-workflow';
 
-export class MissingExecutionStopError extends ApplicationError {
+export class MissingExecutionStopError extends UnexpectedError {
```

Comment:
This is used to stop an execution via endpoint. The Execution ID is coming from the user --> `UserError`